notebooks/Grid Search for Binary Classification.ipynb

```diff
@@ -174,9 +174,11 @@
    "source": [
     "## Reducing Unfairness with Grid Search\n",
     "\n",
-    "Now, we move on to attempting to reduce the unfairness in our model using the grid search. This tries a series of different models, parameterised by a Lagrange multiplier $\\lambda_i$. For each value of $\\lambda$, the algorithm reweights and relabels the input data, and trains a fresh model ($\\lambda=0$ corresponds to the unaltered case).\n",
+    "Now, we move on to attempting to reduce the unfairness in our model using the grid search. This tries a series of different models, parameterized by a Lagrange multiplier $\\lambda_i$. For each value of $\\lambda$, the algorithm reweights and relabels the input data, and trains a fresh model ($\\lambda=0$ corresponds to the unaltered case).\n",
     "\n",
-    "The grid search acts like a normal `sklearn` estimator, implementing `fit()` and `predict()` methods. The `fit()` method performs the grid search, and the best model found (according to a specified selection rule) is used in `predict()` calls. However, after `fit()` is called, there are two extra properties on the estimator - a `best_result` and a list `all_results`; the `best_result` is used by `predict()` while `all_results` corresponds to the output of the grid search itself. The items in each are dictionaries, each with four entries - `lambda_vec`, `objective`, `gamma` and `predictor`.\n",
+    "The grid search acts like a normal `sklearn` estimator, implementing `fit()` and `predict()` methods. The `fit()` method performs the grid search, and the best model found (according to a specified selection rule) is used in `predict()` calls. However, after `fit()` is called, there are several extra properties on the estimator. \n",
+    "- `_best_grid_index` indicates the index of the grid point where grid search found the best solution. The `predict()` method uses the corresponding predictor.\n",
+    "- for each grid point we have an entry in the following four collections: `_lambda_vecs`, `_objectives`, `_gammas`, `_predictors`.\n",
     "\n",
```

Comment:
just switch the order of bullets: first say what we store for each grid point and then which is the best.